lib/defaults.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
   'env': {                           // http://eslint.org/docs/user-guide/configuring.html#specifying-environments
     'browser': true,                 // browser global variables
     'node': true,                    // Node.js global variables and Node.js-specific rules
-    'es6': true
+    'es2020': true
```

Comment:
I don't know if we should do this because I don't think we support transpiling ES2020 code

Reply:
that's why it's a major version change

Reply:
I need this to make eslint work with optional chaining `a?.b`